slaves/linux-cross/arm-linux-gnueabi.config

```diff
@@ -0,0 +1,573 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Crosstool-NG Configuration
+#
+# Generated with crosstool-ng-1.22.0's `ct-ng menuconfig` command using the default configuration
+# plus the following modifications
+#
+# Path and misc options > Prefix directory = /x-tools/${CT_TARGET}
+# Target options > Target Architecture = arm
+# Target options > Architecture level = armv5t -- (*)
```

Comment:
Yeah to be honest the myriad of options we have for compiling to ARM always confuse me, and I basically never know what's what.

We'll run into friction if we diverge from the standard debian toolchains unfortunately, but we also run into friction if we follow debian and not clang. So... either way friction!

In any case, feel free to upgrade this to armv6, but not critical!